src/domain/users/wallets/wallets.repository.interface.ts

```diff
@@ -0,0 +1,5 @@
+export const IWalletsRepository = Symbol('IWalletsRepository');
```

Comment:
Do you think wallet CRUD will occur outside of the user domain? If so, I think we should move it top-level and create a `WalletsRepositoryModule` as well.

Reply:
Good question! I think this is something that still needs to be discussed. As of now I believe Wallet CRUD will only happen in the context of users e.g. creating a user and wallet, adding a wallet to an existing user and removing a wallet from a user. On the other hand User CRUD will happen both on its own e.g. deleting a user and in the context of organisations e.g. adding a member to an org (which possibly needs to create an inactive user).

Reply:
For now I would keep it as part of the user domain as its not planned that wallets will be used independent of users. If that happens at a later stage e.g. if we want to use wallets for notifications then we can still extract it.